makefiles/Makefile.win.mk

```diff
@@ -21,6 +21,10 @@ STATIC_PRE_LIB = $(OR_ROOT)lib\\
 STATIC_POST_LIB = .lib
 O=obj
 E=.exe
+L=.lib
```

Comment:
@lperron Ah, I see. IMO, most folks I work with understand *.lib* to mean static library, which ``L`` is a decent shorthand consistent with ``E``. Which, in Windows anyway, ``DLL`` is always *dynamic link*; except for **nix*, which ``.so`` is for *shared object library*.

That said, thanks for merging! :+1: